docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx

```diff
@@ -117,7 +117,7 @@ The following conventions for specific asset IDs apply to all digital twins:
 
 | Key            | Availability | Description                                                                                  | Type   |
 | :------------- | :----------- | :------------------------------------------------------------------------------------------- | :----- |
-| partInstanceId | Mandatory    | The serial number of the part from the manufacturer.                                         | String |
+| partInstanceId | Mandatory    | The serial number of the part from the part's manufacturer. This must be the serial number that is applied to the part (as barcode or DMC) as the recipient (customer) of the part will use this information to lookup the part's digital twin in the manufacturer's DTR.<br/> A manufacturer might create more than one serial number for a part. For example, if there are customers that require the serial number to have a customer-specific format, the manufacturer might create two serial numbers for every part: an internal one that has a standard format for ease of use in the manufacturer's internal systems and a customer-specific one that is actually applied to the part. Only the customer-specific serial number is used as partInstanceId in this case. | String |
 | intrinsicId    | Mandatory    | This the unified technical key for searching a digital twin of a part. In this case the<br/> partInstanceId is used as the intrinsicId.                                         | String |
```

Comment:
What is meant by: "In this case the<br/> partInstanceId is used as the intrinsicId."
Which "case" is meant here? And in which cases is the intrinsicId to be used?